README.rst

```diff
@@ -205,11 +234,54 @@ run (e.g. ``~/.synapse``), and::
     source ./bin/activate
     synctl start
 
+
+Connecting to Synapse from a client
+===================================
+
+The easiest way to try out your new Synapse installation is by connecting to it
+from a web client. We recommend the one at http://riot.im/app. You will need to
```

Comment:
as matrix.org, we really can't specifically recommend Riot.  i suggest "The easiest option is probably..." or "One option is..."